raiden/transfer/state_change.py

```diff
@@ -1206,6 +1206,10 @@ def __init__(
     def token_network_identifier(self) -> TokenNetworkAddress:
         return TokenNetworkAddress(self.canonical_identifier.token_network_address)
 
+    @property
+    def channel_identifier(self) -> ChannelID:
```

Comment:
purpose and usage?

Reply:
as I explained in description, subdispatch_to_channels_by_participant_address needs this